frontend/packages/console-plugin-sdk/src/codegen/__tests__/plugin-resolver.spec.ts

```diff
@@ -1,33 +1,43 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as findUp from 'find-up';
 import * as readPkg from 'read-pkg';
 import {
   Package,
   PluginPackage,
   isPluginPackage,
   readPackages,
   filterActivePluginPackages,
+  getMonorepoRootDir,
 } from '../plugin-resolver';
 
-export const templatePackage: Package = Object.freeze({
-  name: 'test',
-  version: '0.0.0',
-  readme: '',
-  _id: '@',
-});
+export const getTemplatePackage = ({
+  name = 'test',
+  version = '0.0.0',
+  _path = '/test/packages/test-pkg',
+} = {}): Package =>
+  Object.freeze({
+    name,
+    version,
+    readme: '',
+    _id: `${name}@${version}`,
```

Comment:
I cannot find where `_id` is being used.

Reply:
@christianvogt , hi, thank you for the review, I think it comes from https://github.com/npm/normalize-package-data.

```
# ./build.sh
...
ERROR in /home/yzamir/go/src/github.com/openshift/console/frontend/packages/console-plugin-sdk/src/codegen/__tests__/plugin-resolver.spec.ts
(10,14): Type 'Readonly<{ name: string; version: string; readme: string; }>' is not assignable to type 'NormalizedPackageJson'.
  Property '_id' is missing in type 'Readonly<{ name: string; version: string; readme: string; }>' but required in type 'Package'.
```

Reply:
Got it. Thanks!

Reply:
@christianvogt Just a note, the leading `_` in the newly added `_path` property is a convention to indicate that this property is computed, similar to the `_id`.